core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -217,18 +217,21 @@ class SpendReportingService(userInfo: UserInfo, dataSource: SlickDataSource, big
       .build()
   }
 
-  private def getQuery(aggregationKeys: Set[SpendReportingAggregationKey], tableName: String): String = {
-    s"""
+  def getQuery(aggregationKeys: Set[SpendReportingAggregationKey], tableName: String, customTimePartitionColumn: Option[String]): String = {
+    // The Broad table uses a view with a different column name.
+    val timePartitionColumn = customTimePartitionColumn.getOrElse("_PARTITIONTIME")
+    val queryClause = s"""
        | SELECT
        |  SUM(cost) as cost,
        |  SUM(IFNULL((SELECT SUM(c.amount) FROM UNNEST(credits) c), 0)) as credits,
        |  currency ${aggregationKeys.map(_.bigQueryAliasClause()).mkString}
-       | FROM `$tableName`
```

Comment:
I don't know where back ticks were used-- any reason @marctalbott?

Reply:
Does it work without back ticks? Seems to be pretty standard to wrap the table name in back ticks both in our code and in [Google's documentation](https://cloud.google.com/bigquery/docs/running-queries#queries) (see Java section of example)

Reply:
Yes, it works without back ticks, but I will add it back in.